OpenRA.Game/Sound/Sound.cs

```diff
@@ -119,6 +119,8 @@ static ISound Play(Player player, string name, bool headRelative, WPos pos, floa
 		public static ISound Play(string name, WPos pos, float volumeModifier) { return Play(null, name, false, pos, volumeModifier); }
 		public static ISound PlayToPlayer(Player player, string name) { return Play(player, name, true, WPos.Zero, 1); }
 		public static ISound PlayToPlayer(Player player, string name, WPos pos) { return Play(player, name, false, pos, 1); }
+		public static ISound PlayLooped(string name, WPos pos) { return Play(null, name, true, pos, 1, true); }
+		public static ISound PlayLooped(string name) { return PlayLooped(name, WPos.Zero); }
```

Comment:
This should probably be before the other `PlayLooped` just to be a bit more consistent.